packages/bako-safe/src/BakoSafeConnector.tsx

```diff
@@ -0,0 +1,240 @@
+import axios, { type AxiosInstance } from 'axios';
+import {
+  type Asset,
+  FuelConnector,
+  FuelConnectorEventTypes,
+  type JsonAbi,
+  Provider,
+  type TransactionRequestLike,
+} from 'fuels';
+import { type Socket, io } from 'socket.io-client';
+
+import { DAppWindow } from './DAPPWindow';
+import {
+  API_URL,
+  APP_BSAFE_URL,
+  APP_DESCRIPTION,
+  APP_IMAGE_DARK,
+  APP_IMAGE_LIGHT,
+  APP_NAME,
+} from './constants';
+import { BAKOSAFEConnectorEvents } from './types';
+
+type FuelABI = JsonAbi;
+type Network = {
+  url: string;
+  chainId: number;
+};
```

Comment:
You should be able to import this types from `fuels`. https://github.com/FuelLabs/fuel-connectors/blob/bd56a984271889feca64810b2199741662917e68/packages/fuel-wallet/src/FuelWalletConnector.ts#L13